docs/source/usersguide/settings.rst

```diff
@@ -235,6 +235,101 @@ file in your build directory. Setting the :attr:`openmc.Source.library`
 attribute to the path of this shared library will indicate that it should be
 used for sampling source particles at runtime.
 
+.. _serialized_custom_source:
+
+Custom Serialized Sources
```

Comment:
Makes sense. The naming is a hang on from the first pass at this where this was being serialized to a file, but parameterized is a simpler concept in this case (interestingly since this now only reads the constructor once there's nothing to prevent a user having the 'parameter' be the file to read the serialized form of the source from).